services/injection_epc/injection_dossier.py

```diff
@@ -305,13 +305,49 @@ def recuperer_donnees(cls, admission: BaseAdmission):
                 admission_specifique=admission_generale or admission_doctorat
             ) if admission_generale or admission_doctorat else None,
             "adresses": cls._get_adresses(adresses=adresses),
+            "equivalence": cls._get_equivalence(
+                admission=admission,
+                admission_generale=admission_generale,
+                etudes_secondaires=etudes_secondaires,
+            ) if admission_generale else None,
             "documents": (
                 InjectionEPCCurriculum._recuperer_documents(admission_generale or admission_iufc or admission_doctorat)
                 + documents_specifiques
             ),
             "documents_manquants": cls._recuperer_documents_manquants(admission=admission),
         }
 
+    @classmethod
+    def _get_equivalence(
+        cls,
+        admission: BaseAdmission,
+        admission_generale: GeneralEducationAdmission,
+        etudes_secondaires: Dict[str, str]
+    ):
+        experience_academique = admission.admissioneducationalvaluatedexperiences_set.filter(
+            is_access_title=True,
+        ).annotate(
+            annee_max=Max('educationalexperience__educationalexperienceyear__academic_year__year'),
+        ).order_by('-annee_max').first()
+        experience_annuelle = experience_academique.educationalexperienceyear_set.filter(
+            academic_year__year=experience_academique.annee_max
+        ).first() if experience_academique else None
+        condition_acces_uuid = (
+            etudes_secondaires['osis_uuid']
+            if admission.are_secondary_studies_access_title
+            else (experience_annuelle.uuid if experience_annuelle else None)
+        )
```

Comment:
Est ce que c'est pas mieux de mettre un if pour pas faire les querysets si admission.are_secondary_studies_access_title ? 

```suggestion
        condition_acces_uuid = None
        if admission.are_secondary_studies_access_title:
            condition_acces_uuid = etudes_secondaires['osis_uuid']
        else:
            experience_academique = admission.admissioneducationalvaluatedexperiences_set.filter(
                is_access_title=True,
            ).annotate(
                annee_max=Max('educationalexperience__educationalexperienceyear__academic_year__year'),
            ).order_by('-annee_max').first()
            experience_annuelle = experience_academique.educationalexperienceyear_set.filter(
                academic_year__year=experience_academique.annee_max
            ).first() if experience_academique else None
            condition_acces_uuid = experience_annuelle.uuid if experience_annuelle else None
```